### DISCUSSION TITLE
What is your favorite Text Editor?? (好きな、もしくは使ったことのあるテキストエディタは何ですか?)

### DISCUSSION THREAD

**Discussion by ElefunAi** [NAIST-SE/AttractingContributors#178] [General]
Please help me to add some of your favorite (or you have experience using) Text Editor into my list.
The URL shows below.

https://github.com/ElefunAi/AttractingContributors-texteditor

**Comment by YPanY** [comment]
added Pycharm, pls help me by adding your favorite programming language or framework here👇 THX!
https://github.com/Pan-NLP-NAIST/AttractingContributors-programming_language/blob/main/README.md

**Reply by ElefunAi** [reply]
I have added App list !
Thank you !

**Comment by tktkbohshi** [comment]
added!
plz help me...
https://github.com/tktkbohshi/AttractingContributors-favorite-niconico-douga

**Reply by ElefunAi** [reply]
I have added niconico-douga list !
Thank you !

**Comment by veekay7** [comment]
Added.
Please help me.
https://github.com/VantelinLuca/AttractingContributors-WishCountry

**Reply by ElefunAi** [reply]
I have added Country list !
Thank you !
